tests/fixtures/unicode-set.js

```diff
@@ -2,10 +2,25 @@ const TRANSFORM_U = { unicodeFlag: 'transform', unicodeSetsFlag: 'transform' };
 
 const Basic_Emoji = {
 	get all() { return `${this.strings}|[${this.chars}]` },
+	// node -e "process.stdout.write(require('regenerate-unicode-properties/Property_of_Strings/Basic_Emoji').strings.sort((a,b)=>b.length-a.length).join('|'))"
```

Comment:
Provided node oneliners to update the test fixtures. Personally I piped the result to `pbcopy` and paste directly in the IDE. At least we can do it semi-manually.

We also restructured the RGI_Emoji tests so that we can easily update the result every year.